runtime/src/bank.rs

```diff
@@ -977,6 +977,21 @@ impl Bank {
         (bank_arc, bank_fork)
     }
 
+    #[cfg(feature = "dev-context-only-utils")]
+    pub fn new_from_parent_with_bank_forks(
```

Comment:
Let's not add this to `Bank` as a constructor. This function is using public methods from `Bank` and `BankForks`. So it can be easily implemented as part of the tests.

The test code can carry this function as a utility function. We can copy the function to different test files (in case crate boundary is causing it to be inaccessible).

Reply:
Generally we should avoid repeating the code. In this particular case, adding the constructor to `Bank` seems incorrect, since it's a test-only code. And, it's two lines of code that calls the public functions. So it seems ok to copy the function to tests.

Reply:
I repeated the function. Let me know what you think.